src/Stratis.Bitcoin/Consensus/ChainedHeaderTree.cs

```diff
@@ -1102,19 +1105,19 @@ public override string ToString()
         }
 
         /// <summary>
-        /// Convert the <see cref="DownloadFrom"/> and <see cref="DownloadTo"/> to a list of consecutive hashes where the fist item in the list is the hash of the download from.
+        /// Convert the <see cref="DownloadFrom"/> and <see cref="DownloadTo"/> to a list of consecutive hashes where the first item in the list is the hash of <see cref="DownloadFrom"/>.
```

Comment:
No longer it gives faster list of consecutive hashes. Newly it gives us a list of consecutive chained headers